cea/datamanagement/format_helper/format_helper.py

```
@@ -0,0 +1,99 @@
+"""
+Main script of the formate helper that activates the verification and migration as needed.
+
+"""
+
+import cea.config
+import os
+import subprocess
```

Ruff (0.8.2)

`subprocess` imported but unused

Remove unused import: `subprocess`

(F401)